requirements.txt

```diff
@@ -1,9 +1,9 @@
-aiofiles>=0.8.0
-async_lru~=1.0
-galaxy_importer>=0.4.5
-GitPython>=3.1.24
-jsonschema~=4.4
-packaging
-pulpcore>=3.18
-PyYAML
-semantic_version
+aiofiles>=0.8.0,<0.9
+async_lru>=1.0,<1.1
+galaxy_importer>=0.4.5,<0.5
+GitPython>=3.1.24,<3.2
+jsonschema>=4.4,<4.7
+packaging>=21,<22
```

Comment:
What motivated the lower bound here?

Reply:
Going to have to be more specific with the package name. `packaging`'s changelog hasn't updated much from version 21.0 released last July.